segments/helpers.py

```diff
@@ -249,4 +249,4 @@ def execute_raw_user_query(self, sql):
                     if self.is_valid_member_id(row[0]):
                         yield row[0]
                     else:
-                        logger.exception(f"Query returned invalid result: {row[0]}")
+                        logger.exception(f"Invalid result for sql query:\n{sql}")
```

Comment:
I'm not sure if we should be printing our data modeling to the telemetry either.

Reply:
1. Why?
2. It's the analytics tables which don't reflect our django Database

Reply:
My suggestion is to make the error: `Segment returned an invalid member id.` with an extra kwarg passing in the segment id in question? Also, not your fault, but technically `logger.exception` is only supposed to be used in an exception handler. Maybe we change this to `logger.error`?